fftvis/simulate.py

```diff
@@ -277,15 +284,22 @@ def simulate(
 
             for fi in range(nfreqs):
                 # Compute uv coordinates
-                u, v = (
+                u, v, w = (
```

Comment:
this seems not to be based off the numpy2 PR I made yesterday?

Reply:
You're right, that's weird. I've changed this function to be consistent with the numpy2 updates